src/Paystack.php

```diff
@@ -546,6 +546,23 @@ public function fetchSubscription($subscription_id)
         return $this->setHttpResponse('/subscription/'.$subscription_id, 'GET', [])->getResponse();
     }
 
+    /**
+     * Charge returning customers
+     * Used for reccurring charges
+     */
+    public function chargeAuthorizedCard()
+    {
+        $data = [
+            "authorization_code" => request()->name,
```

Comment:
Why does name map to the authorization code?

Reply:
Oh so sorry....i'll fix it asap